Gruntfile.coffee

```diff
@@ -255,6 +255,7 @@ module.exports = (grunt) ->
   grunt.registerTask "build_src", [ "coffee:src", "emberTemplates:src", "neuter:src" ]
   grunt.registerTask "build_app", [ "replace:app", "coffee:app", "emberTemplates:app", "neuter:app", "copy:app", "less:app" ]
   grunt.registerTask "build_tests", [ "coffee:tests" ]
+  grunt.registerTask "run_tests", [ "coffee:tests", "karma:unit"]
```

Comment:
I think I'll make run_unit_tests and run_functional_tests just in case we ever actually do make functional tests ^^